Problem w/ activity file: 68830_EU_C010.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_EU_C010.html'


Problem w/ activity file: 68830_EU_C011.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_EU_C011.html'


Problem w/ activity file: 68830_NE_C010.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C010.html'


Problem w/ activity file: 68830_NE_C011.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C011.html'


Problem w/ activity file: 68830_NE_C012.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C012.html'


Problem w/ activity file: 68830_NE_C013.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C013.html'


Problem w/ activity file: 68830_NE_C014.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C014.html'


Problem w/ activity file: 68830_NE_C015.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C015.html'


Problem w/ activity file: 68830_NE_C016.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C016.html'


Problem w/ activity file: 68830_NE_C017.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C017.html'


Problem w/ activity file: 68830_NE_C018.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C018.html'


Problem w/ activity file: 68830_NE_C019.html Exception: Traceback (most recent call last):
  File "../../bin/mcq_export2paf.py", line 101, in readActivityConfig
    with open(self.fileName, "rt", encoding="utf-8") as f:
IOError: [Errno 2] No such file or directory: '68830_NE_C019.html'


